package com.greatlearning.coreexception;

public class ExceptionBasics {
	public static void main(String[] args) {

		try {
			int result = 10 / 0;
		} catch (ArithmeticException w) {
			// TODO: handle exception
			//e.printStackTrace();
			System.out.println(w.getMessage());
		}

		System.out.println("WELCOME TO GREAT LEARNING");
	}
}
